Build failure when building with ZeroMQ

|Wazuh version|Component|Install type|Install method|Platform|
|---|---|---|---|---|
|4.1.4 | analysisd | Manager/local | install.sh | CentOS 7 |

CZMQ version 2.2.0, ZeroMQ version 3.2.5

Steps to reproduce:

```
$ make deps
$ make USE_ZEROMQ=yes TARGET=local
....
    CC analysisd/output/zeromq.o
analysisd/output/zeromq.c: In function ‘zeromq_output_event’:
analysisd/output/zeromq.c:115:5: error: too few arguments to function ‘Eventinfo_to_jsonstr’
     char *json_alert = Eventinfo_to_jsonstr(lf);
     ^
In file included from analysisd/output/zeromq.c:17:0:
./analysisd/format/to_json.h:17:7: note: declared here
 char *Eventinfo_to_jsonstr(const Eventinfo *lf, bool force_full_log);
       ^
make[1]: *** [analysisd/output/zeromq.o] Error 1
```

--- a/src/analysisd/output/zeromq.c
+++ b/src/analysisd/output/zeromq.c
@@ -112,7 +112,7 @@ void zeromq_output_end()
 #if CZMQ_VERSION_MAJOR == 2
 void zeromq_output_event(const Eventinfo *lf)
 {
-    char *json_alert = Eventinfo_to_jsonstr(lf);
+    char *json_alert = Eventinfo_to_jsonstr(lf, false);
 
     zmsg_t *msg = zmsg_new();
     zmsg_addstr(msg, "ossec.alerts");
@@ -123,7 +123,7 @@ void zeromq_output_event(const Eventinfo *lf)
 #elif ZMQ_VERSION_MAJOR >= 3
 void zeromq_output_event(const Eventinfo *lf)
 {
-    char *json_alert = Eventinfo_to_jsonstr(lf);
+    char *json_alert = Eventinfo_to_jsonstr(lf, false);
 
     zmsg_t *msg = zmsg_new();
     zmsg_addstr(msg, "ossec.alerts");

